texar/__init__.py

```diff
@@ -23,7 +23,7 @@
 # pylint: disable=wildcard-import
 
```

Comment:
Why is this file needed to be changed at all in this PR?

Reply:
This makes conflicts with other PR which is intended to update this file

Reply:
I had to change it because the CI for python 2.7 was failing because of weird issues in imports. Here is the link to CI that failed https://travis-ci.org/asyml/texar/jobs/569081378#L1779.

Reply:
I just checked the master branch is failing with the same CI error - https://travis-ci.org/asyml/texar/jobs/568599445#L1765